### PR TITLE
feat: update cluster crd

### PR DESCRIPTION
**Pull request by wangyizhi1**
#### What type of PR is this?

feat

#### What does this PR do?

update cluster crd

#### Which issue(s) does this PR fix?

Fixes #
#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: pkg/apis/kosmos/v1alpha1/cluster_types.go [hunk 1]
```diff
@@ -24,44 +24,75 @@ type Cluster struct {
...
+
+	// +optional
+	ClusterLinkOptions ClusterLinkOptions `json:"clusterLinkOptions,omitempty"`
+
+	// +optional
+	ClusterTreeOptions ClusterTreeOptions `json:"clusterTreeOptions,omitempty"`
+}
+
+type ClusterStatus struct {
+	ClusterLinkStatus ClusterLinkStatus `json:"clusterLinkStatus,omitempty"`
```
Comment (OrangeBao) [inline pkg/apis/kosmos/v1alpha1/cluster_types.go:45]: where is  the `ClusteTreeStatus`

Comment (wangyizhi1) [reply pkg/apis/kosmos/v1alpha1/cluster_types.go:45]: Currently  ``ClusteTreeStatus`` is empty and can be added if needed later.

**Pull request merged**